skyvern/webeye/scraper/domUtils.js

```diff
@@ -605,7 +605,13 @@ function getSelectOptions(element) {
       text: removeMultipleSpaces(option.textContent),
     });
   }
-  return selectOptions;
+
+  const selectedOption = element.querySelector("option:checked");
+  if (!selectedOption) {
+    return [selectOptions, ""];
+  }
+
+  return [selectOptions, removeMultipleSpaces(selectedOption.textContent)];
```

Comment:
is the selected value always textContent? could it be the "value" attribute of the element?

Reply:
because when we parse the option. We only use the textContext to represent the value. I think we should be consistent with the logic we do in the option parsing.